build.gradle

```diff
@@ -0,0 +1,51 @@
+plugins {

```

Comment:
I don't see shading setup anywhere. You need it to compile-in external dependencies like minimessage, cloud etc.

Example (note - this is from my kotlin gradle scripts as opposed to groovy, so syntax might differ):
```kotlin
import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
import com.github.jengelman.gradle.plugins.shadow.tasks.ConfigureShadowRelocation
plugins {
  id("com.github.johnrengelman.shadow") version "7.0.0"
}
tasks {
    shadowJar {
        dependsOn(named("relocateShadowJar"))
        minimize()
        archiveClassifier.set("")
    }
    build {
        dependsOn(shadowJar)
    }
}

task<ConfigureShadowRelocation>("relocateShadowJar") {
    target = tasks.shadowJar.get()
    prefix = "com.nearvanilla.deathdb.lib"
}
```

Reply:
`minimize()` broke jetbrains exposed, apparently it's a known issue with shading and jdbc is supposed to fix the issue I was seeing but the recent commit is how I got it to work while still shading other dependencies.